doc/source/placement-group.rst

```diff
@@ -0,0 +1,140 @@
+Placement Group
+===============
+
+Ray is great at scheduling tasks and actors based on its resource requirements.
+requirements, it is also important to specify which "place" they should be scheduled. Ray's placement group APIs allow you to place your actors or tasks according to your strategies.
+
+Let's see some scenarios it is useful.
+
+- **Gang scheduling**: Let's say you'd like to place exact number of actors on each GPU node to maximize bandwidth.
+- **Maximizing data locality**: To avoid Ray objects transfer cost, you'd like to pack all actors or tasks as much as possible.
```

Comment:
I’d imagine data locality means more like “data is stagnant/on ebs and compute is placed next to data”.